driver-core/src/main/java/com/datastax/driver/core/HostConnectionPool.java

```diff
@@ -388,7 +388,12 @@ private Connection findLeastBusyForShard(int shardId) {
   }
 
   ListenableFuture<Connection> borrowConnection(
-      long timeout, TimeUnit unit, int maxQueueSize, ByteBuffer routingKey) {
+      long timeout,
```

Comment:
This is the wrong style. The project does not break lines like this.

Reply:
The Maven project has `fmt-maven-plugin` installed, which checks the style and allows you to automatically format the code (this is how this tool formatted it). I will try to reformat the code, so that both automatic checker and project style are ok.

Reply:
If the automatic checker works then it's ok and don't mind my comment.